sql/table_test.go

```diff
@@ -168,7 +175,9 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		}
 		schema, err := makeTableDesc(stmt[0].(*parser.CreateTable))
 		if err != nil {
```

Comment:
It is being polluted. These changes are only necessary because the test cases here are malformed to begin with. I'd like to see those cases fixed rather than papering over the failures with an exemption for missing primary keys.